cri/v1alpha1/cri.go

```diff
@@ -394,6 +397,7 @@ func (c *CriManager) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 		if err != nil {
 			return nil, fmt.Errorf("failed to remove container %q of sandbox %q: %v", container.ID, podSandboxID, err)
 		}
+		logrus.Infof("success remove container %q of sandbox %q", container.ID, podSandboxID)
```

Comment:
Same issue. Do you think so?

Reply:
Hmm... I think maybe `Infof` is more suitable

The operation of `stop` and `remove` is not as frequent as list

It's helpful and important for us to know details of every pod related operation 😂

Reply:
OK， just double check.